cradle-core/build.gradle

```diff
@@ -1,11 +1,15 @@
 dependencies {
-	compile "org.slf4j:slf4j-api:${slf4j_version}"
+	implementation 'org.testng:testng:7.1.0'
+    compile "org.slf4j:slf4j-api:${slf4j_version}"

```

Comment:
Let's use the same predicate everywhere, i.e. replace "compile" with "implementation" and "testCompile" with "testImplementation".